modules/apps/frontend-js/frontend-js-walkthrough-web/src/main/resources/META-INF/resources/Walkthrough.js

```diff
@@ -409,6 +386,37 @@ const Walkthrough = ({closeOnClickOutside, closeable, skippable, steps}) => {
 	);
 };
 
+const Walkthrough = ({
+	closeOnClickOutside,
+	closeable,
+	pages,
+	skippable,
+	steps,
+}) => {
+	const [
+		currentStepIndex,
+		setCurrentStepIndex,
+	] = useLocalStorage('walkthrough-current-step', () =>
```

Comment:
nit das galáxias aqui: what about creating a constant? :p

Reply:
I'm not sure, it's a little weird to create a variable for this case where I only need to use it in one place, I think if I were using it in more than one place it might make sense just to avoid typo error but in this scenario, it just seems like we're allocating a string in a variable unnecessarily.